data/resources/data/UILayout.json

```diff
@@ -257,7 +257,7 @@
                 "Action": "ToggleVisibilityOfGroup",
                 "ActionParameter": "SettingsMenu",
                 "GroupID": 2,
-                "LayoutGroup": "PauseMenuButtons"
+                "LayoutGroup": "PauseMenuButtons",

```

Comment:
this looks like a syntax error with the , at the end

Have you tried to start cytopia with this? (you probably need to copyresources again to see your changes)

Reply:
Sorry, old code